cmd/kops/update_cluster.go

```diff
@@ -206,6 +200,34 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 		}
 	}
 
+	lifecycleOverrideMap := make(map[cloudup.Phase]fi.Lifecycle)
+
+	for _, override := range c.PhaseOverrides {
+
+		values := strings.Split(override, "=")
```

Comment:
SplitN I think (or validate that len(values) == 2)

Reply:
SplitN I do not think will help, validating len